lib/commands.js

```diff
@@ -292,7 +294,23 @@ var operations = module.exports.operations = {
       if (err) return context.abort();
       Object.keys(masterConfigJSON).forEach(function(key) {
         if (context.oldParameters.hasOwnProperty(key)) {
-          context.oldParameters[key] = masterConfigJSON[key];
+          if (context.oldParameters[key].indexOf('secure') > -1) {
+            var kms = new AWS.KMS({
+              region: context.overrides.templateOptions.AWS_DEFAULT_REGION,
```

Comment:
Yeah this is definitely dubious. The presence of `.overrides.templateOptions` even existing depends on whether or not you passed it in with `overrides` [when calling commands.update() in the first place](https://github.com/mapbox/cfn-config/blob/be0f679075e6526073db2b4bc2caa58baf10fcd4/lib/commands.js#L113). 

Instead, there should be [a `config` object in scope](https://github.com/mapbox/cfn-config/blob/be0f679075e6526073db2b4bc2caa58baf10fcd4/lib/commands.js#L54) with a `.region` property that indicates the region in which the update is being performed.